src/test/clojure/xtdb/node_test.clj

```diff
@@ -259,8 +259,8 @@ WHERE foo.xt$id = 1"]])]
       (t/is (= {{:xt/id 1, :v 1} 2, {:xt/id 1, :v 2} 1}
                (q2 {:basis {:at-tx tx2}, :default-all-valid-time? true}))))))
 
-(t/deftest test-error-handling-inserting-strings-into-app-time-cols-397
```

Comment:
(See #2252)

By my understanding, this test relied on inserting a string to an app time column failing mid transaction. Post our changes - this now fails at the point of running `submit-tx` due to attempting and failing to cast the string to a zoned date time. Updated the test to instead insert an integer - seems to have previous behavior, testing transaction rollback (I believe).